server/filters/formatMultilineTextFilter.ts

```diff
@@ -0,0 +1,7 @@
+export default function formatMultilineTextFilter(value: string): string {
+  if (!value) {
+    return undefined
+  }
+
+  return value.replace(/\n/gm, '<br />')
+}
```

Comment:
Its a really really simple filter - take the text and go a global regex search and replace on `\n`, replacing it with the HTML `<br />`

Reply:
Regex checks out 👍🏻